tests/env/test_env.py

```diff
@@ -63,6 +63,21 @@ def test_env_repr_and_str_when_loaded_from_file(tmp_directory, cleanup_env,
     assert str(env) == str(d)
 
 
+def test_env_load_env_vars(tmp_directory, cleanup_env,
```

Comment:
please set some env vars using monkeypatch: https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest.MonkeyPatch.setenv

and assert that they contain the appropriate values